tools/refresh-keyrings.sh

```diff
@@ -45,11 +30,47 @@ ESM_KEY_ID_BIONIC="2926E7D347A1955504000A983121D2531EF59819"
 # fips and fips-updates are same key ID
 FIPS_KEY_ID_XENIAL="E23341B2A1467EDBF07057D6C1997C40EDE22758"
 
+generate_keyrings() {
+    KEYRING_DIR="$1"
+    shift
+    KEYS="$@"
+
+    # Intentionally unquoted so GPG gets the keys as separate arguments
+
+    for key in $KEYS; do
```

Comment:
*This is more than a nit, in that I would like to see it addressed, but I'm happy for it to happen in a follow-up PR as this is tooling rather than production code.*

I think using a for loop here doesn't make as much sense now that we need to operate on each key separately. It also serves to have the key ID and the actual name we use for that keyring several lines apart, making it harder to easily review what's going on. Would a series of calls that look something like:

```
generate_keyring fips-xenial E23341B2A1467EDBF07057D6C1997C40EDE22758
```

allow us to capture both pieces of information about a key in a single line? (I think this would also allow us to drop the variable definitions above, as the keyring name is more useful as an indicator of what the key ID is for than the variable name is.)

What do you think?